PWGJE/Tasks/jetShape.cxx

```diff
@@ -513,6 +522,10 @@ struct JetShapeTask {
       registry.fill(HIST("trackEta"), track.eta());
       registry.fill(HIST("trackPhi"), track.phi());
 
+      if (!track.isGlobalTrack()) {
```

Comment:
why is this hardcoded rather than using je track selection like for the other process functions?